extensions/kafka/src/main/java/io/deephaven/kafka/ingest/KafkaIngester.java

```diff
@@ -355,11 +355,16 @@ private boolean pollOnce(final Duration timeout) {
             } catch (Throwable ex) {
                 ++messagesWithErr;
                 log.error().append(logPrefix).append("Exception while processing Kafka message:").append(ex).endl();
+                /*
```

Comment:
Afaict, this code block is never getting hit due to io.deephaven.kafka.KafkaTools.SimpleKafkaRecordConsumer#consume.

Example stack:
```
java.lang.IllegalArgumentException: argument "content" is null
        at com.fasterxml.jackson.databind.ObjectMapper._assertNotNull(ObjectMapper.java:4980)
        at com.fasterxml.jackson.databind.ObjectMapper.readTree(ObjectMapper.java:3217)
        at io.deephaven.kafka.ingest.JsonNodeUtil.makeJsonNode(JsonNodeUtil.java:29)
        at io.deephaven.kafka.KafkaTools.lambda$static$5(KafkaTools.java:2094)
        at io.deephaven.kafka.ingest.KafkaStreamPublisher.consumeRecords(KafkaStreamPublisher.java:270)
        at io.deephaven.kafka.KafkaTools$SimpleKafkaRecordConsumer.consume(KafkaTools.java:2302)
        at io.deephaven.kafka.ingest.KafkaIngester.pollOnce(KafkaIngester.java:354)
        at io.deephaven.kafka.ingest.KafkaIngester.consumerLoop(KafkaIngester.java:284)
        at java.lang.Thread.run(Thread.java:829)
```

Reply:
Noted in the new issue.